src/main/java/com/loofah/graph/api/config/DatabaseSeeder.java

```diff
@@ -0,0 +1,27 @@
+package com.loofah.graph.api.config;
+
+import com.loofah.graph.api.models.Skill;
+import com.loofah.graph.api.repositories.SkillsRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DatabaseSeeder implements CommandLineRunner {
+
+    private SkillsRepository skillsRepository;
+
+    @Autowired
+    public DatabaseSeeder(SkillsRepository skillsRepository) {
+        this.skillsRepository = skillsRepository;
+    }
+
+    @Override
+    public void run(String... args) throws Exception {
+        skillsRepository.save(Skill.builder().withTitle("title1").withDescription( "description1").build());
```

Comment:
Does this skill need an Id too like the others?

Reply:
Ah, that was a test I forgot to remove, to see what would happen if we don't specify an ID (it auto-generates a unique one). I've added the manual ID-setting back in now for consistency.